src/ElasticsearchPhpHandler.php

```diff
@@ -53,15 +53,19 @@ public function __invoke(array $request)
 
     private function createPsr7Request(array $ringPhpRequest)
     {
+        // fix for uppercase 'Host' array key in elasticsearch-php 5.3.1 and backward compatible
+        // https://github.com/aws/aws-sdk-php/issues/1225
+        $hostKey = array_key_exists('Host', $ringPhpRequest['headers'])? 'Host' : 'host';
```

Comment:
Could you use `isset($ringPhpRequest['headers']['Host'])` instead?

Reply:
Hi @jeskew, thanks for you reply. I was doing tests, and `array_key_exists` it's more fast than `isset`:

This is the result of first test:
```
isset: 0.0016160011291504
array_key_exists: 0.00042319297790527
```
This is a second one:
```
isset: 0.000020980834960938 (2.0980834960938E-5)
array_key_exists: 0.0000050067901611328 (5.0067901611328E-6)
```

But if you want, if you think that is more "clear" and "clean", i can change it for `isset`. I wait your answer.

Sincerely!

Reply:
I find `isset` to be more clear, but if you strongly disagree I won't block this change on it. Just make sure to add a test.

Reply:
No problem for me @jeskew, I did the change. I tested in a production environment, works fine. :)